src/core/job.c

```diff
@@ -1160,6 +1162,8 @@ int job_serialize(Job *j, FILE *f) {
 
         bus_track_serialize(j->bus_track, f, "subscribed");
 
+        (void) serialize_item(f, "trigger_reason", j->trigger_reason);
```

Comment:
all our serialization keys use `-` instead of `_` for separating words in the name so far